src/NKSolver/NKSolvers.F90

```diff
@@ -1658,9 +1669,9 @@ module ANKSolver
   real(kind=realType) :: ANK_CFLMin0, ANK_CFLMin, ANK_CFLMinBase, ANK_CFLExponent
   real(kind=realType) :: ANK_stepMin, ANK_StepFactor, ANK_constCFLStep
   real(kind=realType) :: ANK_secondOrdSwitchTol, ANK_coupledSwitchTol
-  real(kind=realType) :: ANK_physLSTol, ANK_unstdyLSTol
+  real(kind=realtype) :: ANK_physLSTol, ANK_unstdyLSTol
```

Comment:
Just taking a quick look at this PR. Is there a reason for this change? (And similarly two lines below)

Reply:
no, `realType` to `realtype` changes are just leftovers from me copy pasting. I think I tried setting them to `alwaysRealType` at some point but reverted back to `realType`. I can fix them back to the usual capitalization